app/components/specific/elevationProfile/ElevationProfile.js

```diff
@@ -162,7 +152,7 @@ export default class ElevationProfile extends Component {
               d={area.path}
               stroke={area.color}
               fill={area.color}
-              strokeWidth={1}
+              strokeWidth={0.2}
```

Comment:
lol issue is less visible like that !! no?

Reply:
Smart, isn't it! 😂🤣